VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/EstimatedDecisionDate/EstimatedDecisionDate.test.tsx

```diff
@@ -1,64 +1,42 @@
 import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import { act, ReactTestInstance } from 'react-test-renderer'
 
+import { screen, fireEvent } from '@testing-library/react-native'
 import { DateTime } from 'luxon'
```

Comment:
Can be grouped with the imports at the top

Reply:
changed